core/templates/dev/head/css/oppia.css

```diff
@@ -4558,3 +4558,9 @@ md-card.preview-conversation-skin-supplemental-card {
   padding: 0;
   width: 1px;
 }
+
+
+.popover.bottom.fade.in {
```

Comment:
This looks like it is too broadly scoped. If you want it to apply only to the navbar popover (and not to any other popovers) then you might need to add some parent element classes at the front in order to scope it further.